export const errorFn = () => {
    throw new Error('This is an error function')
}
